### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by ndarilek** [meteor-velocity/html-reporter#71]
As a blind web developer, I've noticed a few accessibility issues:

 * There are 2 unlabeled buttons. Adding aria-label attributes to these should fix them, for instance:
<button class="velocity-btn-close display-toggle" data-target="velocityOverlay" aria-label="Close"></button>
 * I gather from the Velocity website that I'm supposed to click on a green dot to activate the testing overlay, but I'm not sure where this green dot is defined. Is it perhaps an icon displaying whether tests are passing or failing? If so, the <i> should also have an aria-label that indicates status. It should also respond to enter, and perhaps even have tabindex so it is keyboard-focusable.
 * The overlay always appears open with my screen reader active. To fix this, set the aria-hidden attribute on the overlay's <div> to true/false depending on whether the overlay should be hidden.

If I get clarification on the green dot issue, I can try to submit a PR, but my main development platform is Windows so testing will be difficult. But it would also be nice if web developers used ARIA when building tools like these so I'm not perpetually chasing down inaccessible code, so it might also be wise for a core contributor to implement these fixes to learn how they work. Then, hopefully, ARIA will be considered when it comes time to implement additional functionality, and can be baked in from the start. :)

Thanks.

**Comment by samhatoum** [comment]
Thank you for taking interest in Velocity! I've implemented all the fixes you've just mentioned and they are released with the latest version. Please let me know if there's anything else you need done.

The tabindex is 1 by default, but you can also set that in the settings file if you like. Please refer the readme for the deets
